### ISSUE TITLE
Typo in Windows-Matlab installation instructions (?) 

### ISSUE THREAD

**Issue by jonmatthis** [kyamagu/mexopencv#439]
In the CMake instructions for installing for Windows/Matlab, you say the build location should be "C:\dev\build\install" but the attached images suggest that you actually set the build location to be "C:\dev\opencv\build\install"

I think that is a typo? I could also be missing something, as this world is all new to me. 

In any case - Thank you very much for all of your work on this code and documentation!!
_____________________________________________
_____________________________________________

![mexopencv](https://user-images.githubusercontent.com/15314521/50367391-86905900-0545-11e9-862c-1615d3316c80.PNG)

**Comment by amroamroamro** [comment]
Yes if you followed the instructions in the wiki, the correct location is `C:\dev\build\install`.

Basically you should have the following directory structure:

![pic](https://camo.githubusercontent.com/aa827f4f0c264426c40a61677e6ade19a3416146/687474703a2f2f692e696d6775722e636f6d2f637947717651492e706e67)

The images were taken a while ago, and were never updated to match the current guide.
I kept them for illustration purposes :)

**Comment by jonmatthis** [comment]
Thank you!

By the way, while I am noting minor issues - 

The make.m step fails when using the VisualStudio 2017 compiler because of the '-o' flags in the code. After a fair bit of painful googling, I found that replacing all instances of '-o' with '-output' fixes the problem (and throws the warning that the '-output' flag was ignored, so I probably could have just deleted the '-o's'?)
